### PR TITLE
Group release notes by topic

### PR DESCRIPTION
**Pull request by seut**

### CODE REVIEW

File: docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -71,25 +71,24 @@ Deprecations
...
+SQL Statements
+--------------
+
 - Added initial support for cursors. See :ref:`DECLARE <sql-declare>`,
   :ref:`FETCH <sql-fetch>` and :ref:`CLOSE <sql-close>`.
 
-- Added support for ``SET TIME ZONE`` to improve PostgreSQL Compatibility.
-  Timezone will be ignored on the server side.
-
 - Added support for the ``EXISTS`` expression.
```
Comment (matriv) [inline docs/appendices/release-notes/unreleased.rst:80]: Does the order of this and the next one (correlated sub-query) matter and should be switched?

**Review by matriv — APPROVED** [review]
LGTM, left a minor question.

File: docs/appendices/release-notes/unreleased.rst (hunk 1 above)
Comment (seut) [reply docs/appendices/release-notes/unreleased.rst:80]: Why should the order matter in your opinion?

**Comment by seut** [comment]
@matriv I've changed all related entries to use references, please check again.

File: docs/appendices/release-notes/unreleased.rst (hunk 1 above)
Comment (matriv) [reply docs/appendices/release-notes/unreleased.rst]: nevermind, thought that reading top to bottom, sub-queries is a prerequisite for EXISTS.

**Review by matriv — APPROVED** [review]
very nice, thx!

**Pull request merged**